main/settings.py

```diff
@@ -499,16 +499,16 @@
     description="Gdrive folder for video uploads",
     required=False,
 )
-OCW_NEXT_SEARCH_WEBHOOK_KEY = get_string(
-    name="OCW_NEXT_SEARCH_WEBHOOK_KEY",
+OPEN_CATALOG_WEBHOOK_KEY = get_string(
+    name="OPEN_CATALOG_WEBHOOK_KEY",
     default="",
     description="Open discussions webhook key",
     required=False,
 )
-OPEN_DISCUSSIONS_URL = get_string(
-    name="OPEN_DISCUSSIONS_URL",
+OPEN_CATALOG_URLS = get_delimited_list(
```

Comment:
Can we add example values for these new settings in `.env.example`, and document their usage in `README.md`?